test/common/crudtester.go

```diff
@@ -149,11 +149,15 @@ func (c *FederatedTypeCrudTester) createFedResource(adapter federatedtypes.FedAp
 func (c *FederatedTypeCrudTester) CheckCreate(desiredTemplate, desiredPlacement, desiredOverride pkgruntime.Object) (pkgruntime.Object, pkgruntime.Object, pkgruntime.Object) {
 	template, placement, override := c.Create(desiredTemplate, desiredPlacement, desiredOverride)
 
+	overrideResourceVersion := ""
```

Comment:
Good catch! I think the override being optional hasn't been tested in integration and e2e testing is pending @font's join PR.

Reply:
This was also fixed in #14 that isn't merged, except that the log was re-written to not include "override" if the resource does not contain one.